lib/bulk_insert.rb

```diff
@@ -13,15 +13,13 @@ def bulk_insert(*columns, values: nil, set_size:500, ignore: false, update_dupli
           worker.add_all(values)
           worker.save!
         end
-        nil
       elsif block_given?
         transaction do
           yield worker
           worker.save!
         end
-        nil
       else
-        worker
+        []
```

Comment:
i assume that no one in procore is consuming the result of bulk insert, whether using the values, block or neither variant. cause before looks like we used to return the worker and now we return the result

Reply:
This is an outdated diff. Still returning the worker.